### ISSUE TITLE
Enumeration Type Error

### ISSUE THREAD

**Issue by lightumcc** [iewnfod/CAIE_Code#90]
### Discussed in https://github.com/iewnfod/CAIE_Code/discussions/87

<div type='discussions-op-text'>

<sup>Originally posted by **McPhir** April 25, 2024</sup>
For example (like the example in the guide).
I am trying t odo this:
Type Season = (Spring,Summer,Winter,Autumn)
DECLARE ThisSeason : Season
ThisSeason <- Season.Spring
OUTPUT ThisSeason

//not a complaint. The extension is very good.
</div>

**Comment by lightumcc** [comment]
This issue is happened in AST.program.Statements.exe()

```python
try:
                result.append(statement.exe())
            except Exception as e:
                add_python_error_message(str(e), statement)
                break

**Comment by iewnfod** [comment]
fixed in [#67fd2c7](https://github.com/iewnfod/CAIE_Code/commit/67fd2c7ccd2ab407344730beae2ec66da5a167f3)
